language/python/aiohttp_ratelimit.py

```diff
@@ -0,0 +1,103 @@
+# A demo case of async request get with rate limit.
```

Comment:
I updated the catalogue principle just now.
https://github.com/you-n-g/cheatsheets#catalog-principle

Do you think [python_apps](https://github.com/you-n-g/cheatsheets/tree/master/app/python_apps)  will be more suitable for this?

Reply:
LGTM. Make sense to put it in python_apps.